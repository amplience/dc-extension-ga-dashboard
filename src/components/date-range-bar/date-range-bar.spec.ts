import { render } from '@testing-library/svelte';
import DateRangeBar from './date-range-bar.svelte';

describe('DateRangeBar', () => {
  it('should render a DateRangeBar', () => {
    const { container } = render(DateRangeBar, {});
    expect(container.firstChild).toMatchSnapshot();
  });
});
