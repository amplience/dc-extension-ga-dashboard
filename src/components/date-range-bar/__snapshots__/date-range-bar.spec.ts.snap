// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateRangeBar should render a DateRangeBar 1`] = `
<div>
  <section
    class="date-range-bar svelte-e877a4"
  >
    <div
      class="container svelte-e877a4"
    >
      <span
        class="svelte-e877a4"
      >
        Date range
      </span>
       
       
      <section
        class=" svelte-1wu764x"
      >
        <div
          class="date-range-inputs svelte-1wu764x"
          data-testid="date-range-inputs"
        >
          <div
            class="date-selection svelte-1wu764x"
          >
            <div
              class="calendar-icon svelte-zpbvvv"
              style="--icon-width: 24px; --icon-height: 24px;"
            >
              <object
                src="ic-view-calendar.svg/"
              />
            </div>
             
            <input
              class="svelte-1wu764x"
              placeholder="Pick a start date"
              readonly=""
            />
          </div>
           
          <div
            class="date-selection svelte-1wu764x"
          >
            <div
              class="calendar-icon svelte-zpbvvv"
              style="--icon-width: 24px; --icon-height: 24px;"
            >
              <object
                src="ic-view-calendar.svg/"
              />
            </div>
             
            <input
              class="svelte-1wu764x"
              placeholder="Pick an end date"
              readonly=""
            />
          </div>
        </div>
         
      </section>
    </div>
  </section>
</div>
`;
